Assets/_BForBoss/_Core/Scripts/Managers/EnemySpawnerManager.cs

```diff
@@ -1,11 +1,12 @@
-using Perigon.Entities;
+using System;
 using UnityEngine;
 
 namespace BForBoss
 {
     public class EnemySpawnerManager : MonoBehaviour
     {
         private EnemySpawner[] _enemySpawners = null;
+
```

Comment:
instead of doing Null Checks everywhere with `EnemySpawner`, you could set this as an empty array on initialize.